test/feeds.test.js

```diff
@@ -62,3 +62,65 @@ describe('test /feeds/:id responses', () => {
     expect(res.body instanceof Array).toBe(false);
   });
 });
+
+describe('test POST /feeds endpoint', () => {
+  const createdItems = 5;
+  const feedData = {
+    author: 'foo',
+    url: 'http://telescope200.cdot.systems',
```

Comment:
I'm curious about this choice of URL; does this test suite only require a random/broken URL?

Reply:
So this is where I was having issues, if I used 6 here. It would throw a 409 error. Which showed as a duplicate. Yet I was only creating 5? So it confused me.

Reply:
And it just requires a URL that hasn't been added previously.